pkg/agent/util/net_windows.go

```diff
@@ -90,6 +90,22 @@ type Route struct {
 	RouteMetric       int
 }
 
+type AntreaHNSNetwork struct {
```

Comment:
If this type is accessed only inside package `pkg/agent/util`, we can make it private, also for the properties